### PR TITLE
Fix topbar overflowing card for add-on store

### PR DESCRIPTION
**Pull request by rtm516**
## Proposed change

Fixes the installed topbar extending over the rounded corners of the card on the add-on store page

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
N/A
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

Before:
image

After:
image

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by bramkragten — DISMISSED** [review]

File: src/components/ha-card.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export class HaCard extends LitElement {
         display: block;
         transition: all 0.3s ease-out;
         position: relative;
+        overflow: hidden;
```
Comment (bramkragten) [inline src/components/ha-card.ts:30]: this should not be done in `ha-card`, this should be done on the pages using the `ha-card` that are impacted:
```css
ha-card {
    overflow: hidden;
}
```

Comment (rtm516) [reply src/components/ha-card.ts]: Should be sorted

File: hassio/src/dashboard/hassio-addons.ts [hunk 2]
```diff
@@ -125,6 +125,7 @@ class HassioAddons extends LitElement {
       css`
         ha-card {
           cursor: pointer;
+          overflow: hidden;
```
Comment (bramkragten) [inline hassio/src/dashboard/hassio-addons.ts:128]: Also do this for the addon store

**Comment by WRoss7** [comment]
Related to #14277
I didn't make it in time to unstale 😥

**Review by bramkragten — APPROVED** [review]

**Pull request merged**